packages/providers/mgt-msal2-provider/src/Msal2Provider.ts

```diff
@@ -761,6 +762,9 @@ export class Msal2Provider extends IProvider {
             throw popUpErr;
           }
         }
+      } else if (e instanceof BrowserAuthError && e.message.indexOf('no_network_connectivity') > -1) {
```

Comment:
I updated my MGT test application to use a `Login` with a presence:
```
<Login showPresence={true} />
```
Then I quickly reset the network.

I'm seeing a different message as being the cause of the sign out:
![image](https://github.com/plasne/microsoft-graph-toolkit/assets/855217/cc9e2619-0b42-44b1-a04f-fe990526aa27)

Reply:
I can add that one as well.

Reply:
Loading chats retries 3 times. Are you seeing those messages in the logs?

<img width="930" alt="image" src="https://github.com/plasne/microsoft-graph-toolkit/assets/15094473/d89d51eb-7693-4379-a983-b481d801c14e">